docs/core/deploying/native-aot/diagnostics.md

```diff
@@ -0,0 +1,54 @@
+---
+title: AOT Diagnostics
+description: Learn about diagnostics in NativeAOT applications
+author: lakshanf
+ms.author: lakshanf
+ms.date: 08/07/2023
+---
+# Diagnostics in NativeAOT applications
+
+The long term goal for NativeAOT application diagnostics is to provide the rich diagnostic experience that developers expect out of a .NET application. .NET diagnostic experience was built over multiple releases and getting customer feedback on diagnostic was a critical part of that journey. NativeAOT wants to follow a similar path where the right diagnostic experience is built from prioritized customer feedback in multiple releases.
```

Comment:
Existing does use "native AOT" (ie lower case "native" and a space between native and AOT) when referring to NAOT.